Source/Scene/Cesium3DTileStyle.js

```diff
@@ -237,6 +237,8 @@ define([
          * The expression must return or convert to a <code>Boolean</code>.
          * </p>
          *
+         * Applicable to all tile formats.
```

Comment:
To be consistent with the other descriptions, adjust to: `This expression is applicable to all tile formats`
Wrap with `<p> </p>` like the others.
Add this note to `color` as well.